tests/common_fixtures.py

```diff
@@ -493,9 +494,11 @@ def rundb_mock() -> RunDBMock:
     config.dbpath = "http://localhost:12345"
 
     # Create the default project to mimic real MLRun DB (the default project is always available for use):
-    mlrun.get_or_create_project("default")
+    tmp_dir = tempfile.TemporaryDirectory()
```

Comment:
```suggestion
    with tempfile.TemporaryDirectory() as tmp_dir:
```
Will save you the `cleanup()` after the `yield` (and is safer since you don't really catch any exceptions here).